src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -685,52 +785,61 @@ export async function numberOfPositions(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: NumberOfPositionsRequest
-): Promise<string> {
+): Promise<NumberOfPositionsResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const numOfPositions = await panopticish.numberOfPositions(
     wallet
   );
   // TODO: cast this to number/int
-  return numOfPositions;
+  return {
+    numberOfPositions: numOfPositions
+  }
 }
 
 export async function optionPositionBalance(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: OptionPositionBalanceRequest
-): Promise<string> {
+): Promise<OptionsPositionBalanceResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const result = await panopticish.optionPositionBalance(
     wallet,
     req.tokenId
   );
-  // TODO: cast this to number/int
-  return result;
+  return {
+    balance: result.balance,
+    poolUtilization0: result.poolUtilization0, 
+    poolUtilization1: result.poolUtilization1
+  };
 }
 
 export async function pokeMedian(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: PokeMedianRequest
-): Promise<string> {
+): Promise<PokeMedianResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
-  const txData = await panopticish.pokeMedian(wallet);
-  return txData['hash'];
+  const txData = await panopticish.pokeMedian(wallet); 
+  return {
+    other: txData
```

Comment:
What is in this `txData`? If it's a full ethers tx object, let's call this `tx` rather than `other`. If it's something else, then reply to this comment with the object and let's pull off the useful fields and return them in this response.

Reply:
Done: a0b5da895e521e5561b378789bf511424c2297a1